gematria/basic_block/basic_block.cc

```diff
@@ -229,27 +229,63 @@ std::ostream& operator<<(std::ostream& os, const InstructionOperand& operand) {
   return os;
 }
 
+Annotation::Annotation(const std::string& name, double value)
```

Comment:
If we `std::move()`, this should be `std::string name` (no const, no reference). Otherwise, the `std::move()` can't do anything and it must copy the string.